src/components/scene/vr-mode-ui.js

```diff
@@ -71,6 +45,8 @@ module.exports.Component = registerComponent('vr-mode-ui', {
 
     this.orientationModalEl = createOrientationModal(this.exitVR);
     this.el.appendChild(this.orientationModalEl);
+
+    this.attachEventListeners();
```

Comment:
the public API is what is documented. I'm fine with having methods that are not "public" in the prototype. we can flag it as so. and if it's really something we don't want to be called we can make private functions. we already have a bunch of those. Small, easy to understand and testable methods that encourage good hygiene is the priority.